index.js

```diff
@@ -130,21 +189,41 @@ function pdTriggerEvent(group, message, room) {
 }
 
 /**
- * When a refocus.events is dispatch it is handled here.
- *
- * @param {Event} event - The most recent event object
+ * Saves the current list of active rooms to a json file locally
+ * can be read on restart to make active room list persistent.
+ * @returns {Promise} - resolves when data is saved, or error occurs
  */
-function handleEvents(event) {
-  bdk.log.info('Event Activity', event.roomId);
+function updateLocalRoomData() {
```

Comment:
Do we have this practice of saving data into the disk, like this?

Reply:
This is not gonna work when we run this in multiple instances, you are creating a stateful service, no?

Reply:
i followed the same format that trust bot did ? do you think it should be taken out ?

Reply:
Let's keep and please create a story (rank 10) we can triage and put on 226 pipeline same as adding Redis to allow fault tolerance and scalability in bots